packages/cli/src/lib/setup/setupInstall.ts

```diff
@@ -631,22 +629,21 @@ class Install {
                     `host.${hostname} Adapter does not support current os. Required ${adapterConf.common.os}. Actual platform: ${osPlatform}`
                 );
                 return this.processExit(EXIT_CODES.INVALID_OS);
-            } else {
-                if (!adapterConf.common.os.includes(osPlatform)) {
-                    console.error(
-                        `host.${hostname} Adapter does not support current os. Required one of ${adapterConf.common.os.join(
-                            ', '
-                        )}. Actual platform: ${osPlatform}`
-                    );
-                    return this.processExit(EXIT_CODES.INVALID_OS);
-                }
+                // @ts-expect-error yes we want to check if it's in it ;-)
```

Comment:
What was the error? maybe `.includes(osPlatform as any)` would be cleaner if the error is on that call?

Reply:
yep, that's it.
![grafik](https://user-images.githubusercontent.com/17641229/187194572-9351f30e-b083-4939-a947-6e9355e953e5.png)